### DISCUSSION TITLE
[Breadcrumbs] Estilos de texto en link y página activa

### DISCUSSION THREAD

**Discussion by aweell** [Telefonica/mistica-design#623] [Proposals]
Al tratarse las breadcrumbs de una lista de elementos navegables, propongo que se reutilicen los estilos del `text-link` en todos los elementos previos a la página actual y no utilizar un `text-secondary` sino un `text-primary` para el color de texto de la página activa (Puede generar la impresión de que este elemento está deshabilitado?)

Componente actual:
![Screenshot 2022-07-21 at 13 27 44](https://user-images.githubusercontent.com/44420072/180202943-aa157405-f5cb-403b-b722-e637bd845309.png)

https://mistica-n0g2juz14-tuentisre.vercel.app/?path=/story/components-breadcrumbs--default

Propuesta:
![Screenshot 2022-07-21 at 13 28 19](https://user-images.githubusercontent.com/44420072/180203036-14d34820-bfda-4042-896f-4535ded28893.png)

Ejemplo funcional:

https://codepen.io/AlexBueno/pen/KKomGVb

**Comment by aweell** [comment]
Se ha decidido eliminar el código de color del text link en las páginas navegables pero si utilizar la interacción de underline en `:hover` manteniendo el textSecondary en la página activa.

**Comment by yceballost** [comment]
Añado esto por aquí y archivo discussion.
https://github.com/Telefonica/mistica-design/issues/615
